src/lib/link.ts

```diff
@@ -144,6 +189,7 @@ export class Link {
       src.connectionID,
       version
     );
+    console.log('HERE 1');
```

Comment:
To remove before merging. I can add them back when debugging the channel handshake.